### ISSUE TITLE
Let's design a new logo!

### ISSUE THREAD

**Issue by mparvazi** [PrestaShop/hummingbird#248]
Just an idea:

<img src="https://user-images.githubusercontent.com/85633460/166224473-6cc62543-7fc0-4110-a344-d98131a8ff35.png" width="600">

![Screenshot from 2022-05-02 07-02-49](https://user-images.githubusercontent.com/85633460/166224395-53fbe1e4-e9cf-480f-980c-d13a46060301.png)

**Comment by matks** [comment]
🤩 

> Btw we have problem with classic theme.

Soon, it will be deprecated so we dont have to worry a lot 😄

**Comment by NeOMakinG** [comment]
What's the problem with the classic theme?

**Comment by NeOMakinG** [comment]
@PrestaShop/ux-team wdyt?

**Comment by mparvazi** [comment]
> What's the problem with the classic theme?

I think we just can set logo in BO for shop, then it will be used for classic-theme too. right?

**Comment by NeOMakinG** [comment]
> > What's the problem with the classic theme?
> 
> I think we just can set logo in BO for shop, then it will be used for classic-theme too. right?

Seems legit, it's a core issue

**Comment by MatShir** [comment]
Sound a very good idea, we need to check the copyright management before

**Comment by hardstonepaul** [comment]
More ideas
![image](https://user-images.githubusercontent.com/17052773/206317187-1a84e8c1-df8d-4d20-80a3-6ccbabbbc263.png)
With dark and light option.

**Comment by hardstonepaul** [comment]
YES. The pleasure is for me, contribute to this project. thanks for the feedback
![image](https://user-images.githubusercontent.com/17052773/206722028-f514071e-01a8-4eb3-86f5-aa364982f9cd.png)
check new version.

**Comment by NeOMakinG** [comment]
> YES. The pleasure is for me, contribute to this project. thanks for the feedback ![image](https://user-images.githubusercontent.com/17052773/206722028-f514071e-01a8-4eb3-86f5-aa364982f9cd.png) check new version.

Yeahh! Looks damn cool 🎉 wdyt @Hlavtox ?

**Comment by pawelkierklo** [comment]
Do you have this in format SVG ? @hardstonepaul 
Send here :)

**Comment by hardstonepaul** [comment]
Hello Again @pawelkierklo 

Yes of course!!! 🎉

Light Version 
![hummingbird-light-ver](https://github.com/PrestaShop/hummingbird/assets/17052773/eb598d91-2f8e-4492-912d-7a39dba10a31)
And Dark Version
![hummingbird-dark-ver](https://github.com/PrestaShop/hummingbird/assets/17052773/da1178af-3d42-4f1c-9ae3-ff98603d0c27)

Are you going to use it in the new version of the theme?
I feel very proud
Don't hesitate to write to me for anything

> Do you have this in format SVG ? @hardstonepaul Send here :)
